### PR TITLE
E2E test that Queue Full does not break the client. Also that backend exceptions don't break the client or queue.

### PR DESCRIPTION
**Pull request by freddyaboulton**
## Description

Closes: #6713 

Follow up from #6798 

## 🎯 PRs Should Target Issues

Before your create a PR, please check to see if there is [an existing issue](https://github.com/gradio-app/gradio/issues) for this change. If not, please create an issue before you create this PR, unless the fix is very small. 

Not adhering to this guideline will result in the PR being closed. 

## Tests

1. PRs will only be merged if tests pass on CI. To run the tests locally, please set up [your Gradio environment locally](https://github.com/gradio-app/gradio/blob/main/CONTRIBUTING.md) and run the tests: `bash scripts/run_all_tests.sh`

2. You may need to run the linters: `bash scripts/format_backend.sh` and `bash scripts/format_frontend.sh`

### CODE REVIEW

File: js/app/test/queue_full_e2e_test.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import { test, expect } from "@gradio/tootils";
+import { launch_app_background, kill_process } from "./utils";
```
Comment (aliabid94) [inline js/app/test/queue_full_e2e_test.spec.ts:2]: imports not used?

File: js/app/test/queue_full_e2e_test.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+	);
+	const all_text = await Promise.all(all_toast);
+
+	expect(all_text.join("\n")).toContain("This is a gradio error");
+	expect(all_text.join("\n")).toContain("application is too busy");
+
+	await expect
+		.poll(async () => page.getByLabel("First Result").inputValue())
+		.toBeTruthy();
+	await expect
```
Comment (aliabid94) [inline js/app/test/queue_full_e2e_test.spec.ts:27]: why twice?

**Review by aliabid94 — APPROVED** [review]
LGTM

**Comment by freddyaboulton** [comment]
Thanks @aliabid94 !

File: js/app/test/queue_full_e2e_test.spec.ts (hunk 2 above)
Comment (freddyaboulton) [reply js/app/test/queue_full_e2e_test.spec.ts:26]: Typo. Will fix in another PR!

**Pull request merged**